src/planscape/plan/views.py

```diff
@@ -280,9 +282,19 @@ def update_project(request: HttpRequest) -> HttpResponse:
         if project.owner != owner:
             raise ValueError(
                 "You do not have permission to view this project.")
+        
+        max_budget = body.get('max_budget', None)
```

Comment:
nit: I'd prefer seeing 'max_budget', 'max_treatment_area_ratio', etc. as constants defined in a single place

Reply:
I'm not sure what you mean since this is a HTTP method and not a class. Can you clarify?

Reply:
yea:
I mean 'max_budget' is defined in-line in two places, which increases the risk of typos, and if the text is changed, it needs to be changed in two locations.

I'd prefer to say
_MAX_BUDGET_URL_PARAM = 'max_budget' # near top of file
body.get(_MAX_BUDGET_URL_PARAM, None) # in the function

wdyt? I'll approve since this suggestion shouldn't block the pr.

Reply:
oh yes! i see now. i can make that change before merging. thanks for the explanation!